### PR TITLE
adding max_throughput parameter

### PR DESCRIPTION
**Pull request by prashy6191**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -42,3 +42,9 @@ variable "connector_timeout" {
   type        = string
   default     = "10m"
 }
+
+variable "max_throughput" {
+  description = "size of network throughput of the VPC Access Connector"
```
Comment (mfarix) [inline variables.tf]:
```suggestion
  description = "Maximum throughput of the Serverless VPC Access connector in Mbps."
```

File: variables.tf [hunk 2]
```diff
@@ -42,3 +42,9 @@ variable "connector_timeout" {
...
+  type        = number
+  default     = 1000
```
Comment (mfarix) [inline variables.tf]:
```suggestion
  default     = 300
```
Keep default as 300, which is the [default value from provider](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/vpc_access_connector#max_throughput).

File: variables.tf [hunk 3]
```diff
@@ -42,3 +42,9 @@ variable "connector_timeout" {
...
+}
```
Comment (mfarix) [inline variables.tf:50]:
```suggestion
}

```

**Review by mfarix — APPROVED** [review]

**Review by ahmedzidan — APPROVED** [review]
Lgtm

**Pull request merged**